keyboards/tw40/readme.md

```diff
@@ -0,0 +1,17 @@
+# Training Wheel 40
+
+![tw40](https://cdn.shopify.com/s/files/1/0501/7537/7592/products/tw40_600x.jpg?v=1607835284)
+
+* Keyboard Maintainer: [KnoblesseOblige](https://github.com/yourusername)
+* Hardware Supported: Training Wheel 40 PCB
+* Hardware Availability: https://p3dstore.com/products/tw40-gb
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
I've updated the README to include reset instructions. I couldn't find any prior art so hopefully my brief sentence is sufficient.